test/sanity/code-smell/line-endings.sh

```diff
@@ -4,7 +4,9 @@ grep -rIPl '\r' . \
     --exclude-dir .git \
     --exclude-dir .tox \
     | grep -v -F \
-    -e './test/integration/targets/win_regmerge/templates/win_line_ending.j2'
+    -e './test/integration/targets/template/files/foo.dos.txt' \
+    -e './test/integration/targets/win_regmerge/templates/win_line_ending.j2' \
+    -e './test/integration/targets/win_template/files/foo.dos.txt'
```

Comment:
Since there's a blank line following this, it's OK to have a trailing `\` -- it will make it cleaner to add another line after this later. Yes, the original script should have had that already.